projects/sonar/src/environments/environment.prod.ts

```diff
@@ -20,6 +20,7 @@ export const environment = {
   $refPrefix: 'https://sonar.ch',
   globalViewName: 'global',
   languages: ['fr', 'de', 'it', 'en'],
+  projectTitle: 'Swiss Open Access Repository',
```

Comment:
Is translation missing?

Reply:
I added the string marking. This string is translated into the Sonar project.